assets/src/edit-story/components/header/postPublishDialog.js

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { Plain } from '../button';
+import Dialog from '../dialog';
+import Link from '../link';
+
+const Paragraph = styled.p`
+  font-family: ${({ theme }) => theme.fonts.body1.family};
+  font-size: ${({ theme }) => theme.fonts.body1.size};
+  line-height: ${({ theme }) => theme.fonts.body1.lineHeight};
+  letter-spacing: ${({ theme }) => theme.fonts.body1.letterSpacing};
+`;
+
+function PostPublishDialog({ open, onClose, confirmURL, storyURL }) {
+  return (
+    <Dialog
+      open={open}
+      onClose={onClose}
+      title={__('Story published!', 'web-stories')}
+      actions={
+        <>
+          <Plain onClick={onClose}>{__('Dismiss', 'web-stories')}</Plain>
+          <Plain href={confirmURL}>
```

Comment:
I always hesitate to do so because of accessibility, but yeah probably makes sense here.

Reply:
Why would want this to open in a new tab. It is an internal link that user had opts to click. It should not open in a new tab.

Reply:
Because there is no clear navigational path right back to the previous place (the story edit). The only way to get back right away to the created story is by clicking the Back button. Also, it's a completely different context. Might be good to confirm the general rules with the UX though.

Reply:
How is this different from clicking any other link on the page? If I click posts, it would do the same thing?

Reply:
In case of any other link, the user has decided to leave the editor page knowingly as opposed to an action suggested by the editor.

Reply:
You have saved the story at this point, where is the issue with navigating away?

Reply:
Let's leave it like this for now.

According to https://github.com/google/web-stories-wp/issues/1118#issuecomment-642474545 the whole flow will look much different post-beta. We can re-evaluate there.